src/App.tsx

```diff
@@ -1,61 +1,102 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TODO } from './types/todo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [visibleTodos, setVisibleTodos] = useState([...todosFromServer]);
+
+  const [title, setTitle] = useState('');
+  const [currUser, setCurrUser] = useState(0);
+
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const reset = () => {
+    setTitle('');
+    setCurrUser(0);
+  };
+
+  const submit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!title) {
+      setTitleError(true);
+    }
+
+    if (currUser === 0) {
+      setUserError(true);
+    }
+
+    if (!title || !currUser) {
+      return;
+    }
```

Comment:
The error handling logic for `title` and `currUser` could be improved. Currently, the errors are set but not reset if the user corrects the input. Consider resetting `titleError` and `userError` to `false` when the user starts typing or selects a user.